spec/lib/simp/cli/commands/files/el7/incomplete_prev_simp_conf.yaml

```diff
@@ -14,8 +14,6 @@ cli::simp::scenario: simp_lite
 cli::use_internet_simp_yum_repos: true
 cli::version: "0.0.0"
 # NONINTERACTIVE ITEMS TO BE AUTOMATICALLY SET
```

Comment:
This test in part makes sure that non-interactive items are automatically set. Need to find other items that fall into that category, comment them out, and move the comment to them.